.github/workflows/docs.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Install Rust
```

Comment:
Does the mdbook binary actually need rust, or is that for future use?

Reply:
Future